project/frontend/src/ResultsMap.js

```diff
@@ -0,0 +1,30 @@
+import { Map, GoogleApiWrapper } from 'google-maps-react';
+import React from 'react';
+
+const mapStyle = {
+  border: 'thin solid black',
+  height: '500px',
+  width: '500px',
+};
+
+export class MapContainer extends React.Component {
```

Comment:
Lets add aria labels to this as we go: https://reactjs.org/docs/accessibility.html

Reply:
add the aria label to the class ?